src/index.js

```diff
@@ -28,5 +29,6 @@ export default {
     generateThumbnail,
     isValidDeviceName,
     watchWindow,
-    bringWindowToFront
+    bringWindowToFront,
+    getDeviceDatabase
```

Comment:
Maybe let's call it `getDevicesViewportsSet`?

\cc @VasilyStrelyaev

Reply:
maybe `getDevicesViewportData` ?

Reply:
Sounds good